### ISSUE TITLE
494th SOP version numbering

### ISSUE THREAD

**Issue by 132ndNeck** [daviddcs/documentation#6]
I am not sure if I understand the version numbering. Maybe switch to a simpler one, with X.XX

X: big major revision, major changes  (Version 1, Version 2, Version 3)
X.X Minor changes, minor revision (version 1.1, Version 1.2 Vesion 1.3)
X.XX Cosmetic changes: Version 1.11, version 1.12, version 1.13 ) - might not even be needed.

Published version on website, in PDF format will always be the valid version. And the github page is used for work with new versions

**Comment by daviddcs** [comment]
agreed, I'm unsure if github release, versions could be of any help.

**Comment by daviddcs** [comment]
I'm guessing Hamsters numbering was YYMM(version)
